data/src/main/java/me/androidbox/data/mapper/ExtensionMapper.kt

```diff
@@ -1,9 +1,28 @@
 package me.androidbox.data.mapper
 
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.Types
+import me.androidbox.data.local.converter.AttendeeConverter
 import me.androidbox.data.local.entity.EventEntity
 import me.androidbox.domain.authentication.model.Event
 
-fun EventEntity.toEvent(): Event {
+fun List<String>.toJson(): String {
+    val moshi = Moshi.Builder().build()
+    val type = Types.newParameterizedType(List::class.java, String::class.java)
+    val jsonAdapter = moshi.adapter<List<String>>(type)
+
+    return jsonAdapter.toJson(this)
+}
```

Comment:
serialize the list into json

Reply:
Could probably also make this generic so it works with any type of list

Reply:
@philipplackner Create a typeconverter for this, so I have removed all of these now.